packages/pluggableWidgets/app-events-native/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "baseUrl": "./",
     "include": ["node_modules/@mendix/pluggable-widgets-tools/typings", "./src", "./typings"],
     "compilerOptions": {
-        "typeRoots": ["node_modules/@types", "../../../node_modules/@types"]
+        "typeRoots": ["node_modules/@types", "../../../node_modules/@types"],
+        "types": ["jest-image-snapshot"]
```

Comment:
If there are no screenshot tests, then we don't need this compiler option, right 🤔

Reply:
Fixed